### PR TITLE
RHTAPINST-74: README.md

### PR DESCRIPTION
**Pull request by otaviof**
Initial project documentation front page.

### CODE REVIEW

**Comment by prietyc123** [comment]
@otaviof the detailed process of using this cli would be handled via main documentation, correct?

**Review by prietyc123 — CHANGES_REQUESTED** [review]
Can we also add a quick demo section just to let contributor understand how they should start with this?

**Comment by otaviof** [comment]
> @otaviof the detailed process of using this cli would be handled via main documentation, correct?

I'd like to tackle this after we rename a couple of internal subcommands, should happen quite soon, please.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+Red Hat Trusted Application Pipeline Instaler (`rhtap-cli`)
```
Comment (Roming22) [inline README.md]: Instaler -> Installer

Comment (otaviof) [reply README.md]: Thanks!

**Comment by otaviof** [comment]
> Can we also add a quick demo section just to let contributor understand how they should start with this?

Please consider the latest changes, I've added the `CONTRIBUTING.md`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+Contributing to `rhtap-cli`
+---------------------------
+
+In order to contribute to this project you need the following requirements:
+
+- [Golang 1.16 or higher][golang]
+- [GNU Make][gnuMake]
+
+All the automation needed for the project lives in the [Makefile](Makefile). This file is the entry point for all the automation task in the project for CI, development and release
```
Comment (Roming22) [inline CONTRIBUTING.md]: *tasks

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+# Building
+
+After you have cloned the repository and installed the requirements, you can start building. For that purpose you can simply run `make`, the default target builds the application on the `bin` directory
```
Comment (Roming22) [inline CONTRIBUTING.md]: *in the `bin`

**Comment by Roming22** [comment]
Feel free to merge once the typos are fixed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (otaviof) [reply CONTRIBUTING.md]: Thanks!

File: CONTRIBUTING.md (hunk 3 above)
Comment (otaviof) [reply CONTRIBUTING.md]: THanks!

**Pull request merged**